appinfo/info.xml

```diff
@@ -19,7 +19,7 @@ For more information, please read the [ownCloud Documentation](https://doc.owncl
 	<screenshot>https://raw.githubusercontent.com/owncloud/screenshots/master/templateeditor/template_editor.jpg</screenshot>
 	<namespace>TemplateEditor</namespace>
 	<dependencies>
-		<owncloud min-version="10.0.3" max-version="10.1" />
+		<owncloud min-version="10.0.8" max-version="10.1" />
```

Comment:
10.0.9 or is 8 enough ?

Reply:
There is nothing 10.0.9-specific, I added a check for footers so 10.0.8 users will not see common mail footers in the list